radixdlt/babylon-gateway
# feature: Pending transactions rework

Testing performed:
* Submit invalid transaction
* Submit valid transaction. Check DB and status.
* Submit duplicate valid transaction. Check DB and status.
* Submit transaction with down Core API on Gateway, check interim API result and check gets resubmitted by Aggregator. Use DB to check that resubmission sees exponential back-off.
* Adjusted pruning delay and checked it works
